app/controllers/users_controller.rb

```diff
@@ -0,0 +1,39 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :authorize_user, except: [:create, :new, :edit, :update]
+
+  def index
+    @users = User.order("email ASC").page(params[:page])
```

Comment:
We aren't requiring first or last name for users, so email seemed the next logical choice to order by, though we could order by created_at... but that seems to make searching less intuitive